packages/cli/src/commands/org/config/monitor.ts

```diff
@@ -76,11 +76,14 @@ export default class Monitor extends Command {
     cli.action.stop(this.getReportStatus(snapshot.latestReport));
   }
 
-  private printHeader(snapshotId: string) {
+  private printHeader() {
+    const {args} = this.parse(Monitor);
+    const snapshotId = args.snapshotId;
     const header = ReportViewerStyles.header(
-      `\nMonitoring snapshot ${snapshotId}:`
+      `Monitoring snapshot ${snapshotId}`
     );
-    cli.log(header);
+    cli.log('');
```

Comment:
Why the log '' ?

Reply:
Just to create some space between the command and the message.